python/gstaichi/lang/kernel_impl.py

```diff
@@ -1082,6 +1083,22 @@ def recursive_set_args(needed_arg_type: Type, provided_arg_type: Type, v: Any, i
             prog = impl.get_runtime().prog
             if not compiled_kernel_data:
                 compile_result: CompileResult = prog.compile_kernel(prog.config(), prog.get_device_caps(), t_kernel)
+                if os.environ.get("TI_DUMP_KERNEL_CHECKSUMS", "0") == "1":
+                    checksums_file_path = pathlib.Path("/tmp/checksums.csv")
+                    kernels_dump_dir = pathlib.Path("/tmp/kernels")
```

Comment:
At this point we should be able to specify a custom location where to store all these dump via some environment variables. We have been relying on hard-coded path too often already for all these `TS_DUMP_*`.